src/main/java/org/folio/bulkops/batch/jobs/processidentifiers/BulkEditUserIdentifiersJobConfig.java

```diff
@@ -101,6 +103,8 @@ public Step bulkEditUserStep(FlatFileItemReader<ItemIdentifier> csvItemIdentifie
       .processor(bulkEditUserProcessor)
       .faultTolerant()
       .skipLimit(1_000_000)
+      .retry(ConverterException.class)
+      .retryLimit(1_000_000)
```

Comment:
1_000_000 - please, don't forget to analyze and change this number to more reasonable.